build.airunner.linux.prod.spec

```diff
@@ -5,31 +5,16 @@ from PyInstaller.utils.hooks import copy_metadata, collect_data_files
 import sys ; sys.setrecursionlimit(sys.getrecursionlimit() * 5)
 os.environ["AIRUNNER_ENVIRONMENT"] = "prod"
 os.environ["LD_LIBRARY_PATH"] = "/usr/local/lib/python3.10/dist-packages/PyQt6/Qt6/lib/:/usr/lib/x86_64-linux-gnu/wine-development/:/usr/local/lib/python3.10/dist-packages/h5py.libs/:/usr/local/lib/python3.10/dist-packages/scipy.libs/:/usr/local/lib/python3.10/dist-packages/tokenizers.libs/:/usr/local/lib/python3.10/dist-packages/Pillow.libs/:/usr/local/lib/python3.10/dist-packages/opencv_python.libs/:/usr/local/lib/python3.10/dist-packages/torchaudio/lib/:/usr/local/lib/python3.10/dist-packages/torch/lib/:/usr/lib/python3.10:/usr/lib/x86_64-linux-gnu/:/usr/local/lib/:/usr/local/lib/python3.10:/usr/local/lib/python3.10/dist-packages"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/cuda/lib64"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/cuda-11.7/targets/x86_64-linux/lib/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/tensorrt/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/opencv_python.libs/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/lib/x86_64-linux-gnu/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/Pillow.libs/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/tokenizers.libs/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/xformers/triton"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/nvidia/cuda_runtime/lib/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/nvidia/cudnn/lib"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/numpy.libs"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/h5py.libs"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/torchaudio/lib/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/torch/lib/"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/torch/bin"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/torch/_C"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/torch"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/triton"
-os.environ["PATH"] = f"{os.environ['PATH']}:/usr/local/lib/python3.10/dist-packages/triton/_C"
-os.environ["PYTHONPATH"]=f"${os.environ['PYTHONPATH']}:/usr/local/lib/python3.10/dist-packages"
 block_cipher = None
 DEBUGGING = True
-ONE_fILE = False
-ROOT = "/app/airunner"
-DIST = "/app/dist/airunner"
+EXCLUDE_BINARIES = True
```

Comment:
This was set to False which was causing the conflicts we saw in actions - everything else in this PR addresses further issues I ran into after moving past the name conflict